src/createClient.spec.ts

```diff
@@ -236,6 +301,20 @@ describe('useQuery', () => {
     expect(query1.response).toBe(true)
     expect(query2.response).toBe(false)
   })
+
+  test('placeholder', async () => {
```

Comment:
same exact test from query. I wouldn't be opposed to saying, if it passes for query we can assume it also works for useQuery like we do for other tests, but I thought it was worth a convo.